course_catalog/migrations/0003_auto_20190108_2355.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.0.8 on 2019-01-08 23:55
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('course_catalog', '0002_auto_20181221_1825'),
```

Comment:
Also rename this migration script, auto-generated names will fail the tests.

Reply:
👍